ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +20,10 @@ nginx_sites:
    add_header: "X-Frame-Options SAMEORIGIN"
    access_log: "{{ log_home }}/{{ deploy_env }}-timing.log timing"
    locations:
+    - name: "/a/ipm-senegal/phone/restore"
```

Comment:
This looks right. Any thoughts about also doing an HQ PR to check the domain on submission and restores?

Reply:
Good idea, can't be too careful.